esp/templates/program/modules/commmodule/step2.html

```diff
@@ -38,7 +38,8 @@
 Please write your email/message now. Note that you can use <tt>{{ parameter }}</tt> to enter a parameter
 into the text of the message&mdash;the parameters are listed below.
 
-<p><b>Tip:</b> If you would like to send your message in HTML format, wrap your entire message body in &lt;html&gt; and &lt;/html&gt; tags.</p>
+<p><b>Tip:</b> The default {{ settings.ORGANIZATION_SHORT_NAME }} template (which you can modify <a href="/admin/utils/templateoverride/">here</a>) is automatically wrapped around your message below.
+Don't forget to write the body in HTML, but no need for &lt;html&gt; and &lt;/html&gt; tags.</p>
```

Comment:
Okay, cool. I don't know how to safely render emails without making them log in, but I can search around a bit.

Sounds good. I'm not super familiar with how Markdown works, but I'll check out the QSD-conversion code and see if I can figure out how to adapt it.

Good call on the release notes. At this point, I think I should open a new pull request after making some changes to this one.

To summarize, next steps are:
1. Look into making a more general code that will permit admins to enter plain or marked-up text from the comm panel and merge their message with the default HTML template.
2. Expand the current default template to include some "invisible" compatibility code (for phones, and maybe a "view in browser" link) as well as a generic unsubscribe footer and easy slots for admins to add/edit their own headers and footers.
3. Update step2.html (include a link directly to the template override) and release notes with new instructions for admins. This should probably include some warnings about what not to touch (autoescape tags and where not to mix HTML/Markdown)
4. Update Travis if necessary (depending on how steps 1 and 2 work out).
5. Fix style in the Python files (among other things, use <tt>render_to_string()</tt> or find out what went wrong last time).